ground-server/src/app/layout.tsx

```diff
@@ -16,8 +16,8 @@ const geistMono = localFont({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "Ground Server",
+  description: "The Ground Server that tracks and displays telemetry data from the Cornell Rocketry Team's rocket",
```

Comment:
" and sends commands" We'll probably forget to update this later, so might as well make it reflect the final state